src/app/about/page.js

```diff
@@ -1,9 +1,11 @@
 import Banner from "@/components/Banner";
+import Heading from "src/components/Heading.jsx";
 
 const about = () => {
   return (
     <div>
       <Banner />
+      <Heading text="Join HLG@UCR" />
```

Comment:
we can leave this honestly but make the text "Who We Are" cuz it's the about page